### ISSUE TITLE
Unconfirmed bitcoin 

### ISSUE THREAD

**Issue by forbes147** [blockchain/My-Wallet-V3-Android#1317]
How long does blockchain want to confirm my transactions???

**Comment by Apetree100122** [comment]
Usually 2 to confirm and 6 for valid

**Comment by forbes147** [comment]
2


Sent from the all new AOL app for iOS


On Saturday, June 10, 2023, 12:56 PM, Alexander Petree ***@***.***> wrote:




Usually 2 to confirm and 6 for valid

**Comment by forbes147** [comment]
Still unconfirmed 


Sent from the all new AOL app for iOS


On Saturday, June 10, 2023, 12:56 PM, Alexander Petree ***@***.***> wrote:




Usually 2 to confirm and 6 for valid

**Comment by chrisspradling1980** [comment]
Uh-oh...!

**Comment by chrisspradling1980** [comment]
I'm seriously curious about this. Last week I opened my
Blockchain/My-Wallet on my desktop to explore the files. I did not notice
anything out of the ordinary but, I also opened my Bitcoin wallet on my
desktop. Maybe some mining or pooling was accounted for somewhere and we
just don't have a way to track it to the source.
Regards

**Comment by forbes147** [comment]
Me too! I still can’t track mine even with me having the blockchain hash ID, I still can’t track anything. This has been since last year November. 


Sent from the all new AOL app for iOS


On Saturday, June 10, 2023, 9:36 PM, Chris Spradling ***@***.***> wrote:



I'm seriously curious about this. Last week I opened my
Blockchain/My-Wallet on my desktop to explore the files. I did not notice
anything out of the ordinary but, I also opened my Bitcoin wallet on my
desktop. Maybe some mining or pooling was accounted for somewhere and we
just don't have a way to track it to the source.
Regards
